reposerver/repository/repository.go

```diff
@@ -613,6 +611,16 @@ func makeJsonnetVm(sourceJsonnet v1alpha1.ApplicationSourceJsonnet, env *v1alpha
 		}
 	}
 
+	// Jsonnet Imports relative to the repository path
+	jpaths := []string{appPath}
+	for _, p := range sourceJsonnet.Libs {
+		jpaths = append(jpaths, path.Join(repoRoot, p))
```

Comment:
I think this needs to ensure that the result of `path.Join()` does not point outside of `repoRoot`. I.e. now, if `p` is `../foo`, and `repoRoot` is `/tmp/bar`, the resulting jpath would be `/tmp/foo`.

We do not want an application to have access to files outside the checked out repository.

Reply:
Fixed